src/linear_algebra/invariant_basis_number.lean

```diff
@@ -96,6 +97,21 @@ begin
     ((P.injective.comp i).comp Q.injective)
 end
 
+/-- A ring satisfies the strong rank condition if and only if, for all `n : ℕ`, any linear map
+`(fin (n + 1) → R) →ₗ[R] (fin n → R)` is not injective. -/
+lemma strong_rank_condition_iff : strong_rank_condition R ↔
```

Comment:
I think you should maybe make this more specific
```suggestion
lemma strong_rank_condition_iff_succ : strong_rank_condition R ↔
```
and then stick `@[mk_iff]` on `strong_rank_condition` to get the definitional version as `strong_rank_condition_iff, which generates:
```lean
strong_rank_condition_iff :
  ∀ (R : Type u_1) [_inst_1 : ring R],
    strong_rank_condition R ↔ ∀ {n m : ℕ} (f : (fin n → R) →ₗ[R] fin m → R), injective ⇑f → n ≤ m
```